flytekit/core/python_function_task.py

```diff
@@ -244,20 +144,23 @@ def execute(self, **kwargs) -> Any:
             return self.dynamic_execute(self._task_function, **kwargs)
 
     def get_command(self, settings: SerializationSettings) -> List[str]:
-        return [
+        container_args = [
             "pyflyte-execute",
-            "--task-module",
-            self._task_function.__module__,
-            "--task-name",
-            self._task_function.__name__,
             "--inputs",
             "{{.input}}",
             "--output-prefix",
             "{{.outputPrefix}}",
             "--raw-output-data-prefix",
             "{{.rawOutputDataPrefix}}",
+            "--resolver",
+            self.task_resolver.location,
+            "--",
```

Comment:
i think i really prefer --resolver-arg type of usage, why do you want this raw command line?

Reply:
Spent half an hour getting it to work with click and gave up?